### PR TITLE
remove new ghost inspector test from signups table

### PR DESCRIPTION
**Pull request by sohaibhasan**
#### What's this PR do?
* Removes records created by an unaccounted for ghost inspector process from campaign activity 
#### Where should the reviewer start?
* campaign_activity.sql
#### How should this be manually tested?
* Check if signups table has records where `why_participated` = `Testing from Ghost Inspector!`
#### What are the relevant tickets?
* https://www.pivotaltracker.com/story/show/163595754

### CODE REVIEW

File: data/sql/derived-tables/campaign_activity.sql [hunk 1]
```diff
@@ -19,12 +19,13 @@ CREATE MATERIALIZED VIEW public.signups AS
         FROM rogue.signups stemp
         WHERE stemp.deleted_at IS NULL
         AND stemp."source" IS DISTINCT FROM 'runscope'
-	AND stemp."source" IS DISTINCT FROM 'runscope-oauth'
-	AND stemp."source" IS DISTINCT  FROM 'rogue-oauth'
+		AND stemp."source" IS DISTINCT FROM 'runscope-oauth'
+		AND stemp."source" IS DISTINCT  FROM 'rogue-oauth'
         AND stemp.why_participated IS DISTINCT FROM 'why_participated_ghost'
+        AND stemp.why_participated = 'Testing from Ghost Inspector!'
```
Comment (mjain-1) [inline data/sql/derived-tables/campaign_activity.sql]: this should be 'IS DISTINCT FROM'

Comment (sohaibhasan) [reply data/sql/derived-tables/campaign_activity.sql]: YIKES. Just fixed this

Comment (mjain-1) [reply data/sql/derived-tables/campaign_activity.sql]: 👍

**Comment by sohaibhasan** [comment]
Please don't merge this. I will do so after I have flagged the change for mgmt

**Pull request merged**